app/assets/javascripts/discourse/app/lib/uppy/composer-upload.js

```diff
@@ -735,20 +733,35 @@ export default class UppyComposerUpload {
   #autoGridImages() {
     const reply = this.composerModel.get("reply");
     const imagesToWrapGrid = new Set(this.#consecutiveImages);
-    const matches = reply.match(IMAGE_MARKDOWN_REGEX) || [];
+    const uploadingImagePattern = /\[Uploading: ([^\]]+?)\.\w+…\]\(\)/g;
```

Comment:
Does this work with non-English locales? Not sure what we output here for `Processing:` or `Uploading:`.